homeassistant/components/light/isy994.py

```diff
@@ -64,7 +64,7 @@ def turn_off(self, **kwargs) -> None:
         if not self._node.fastoff():
             _LOGGER.debug('Unable to turn on light.')
 
-    def turn_on(self, brightness=100, **kwargs) -> None:
+    def turn_on(self, brightness=255, **kwargs) -> None:
```

Comment:
Switching to `None` works for getting the dimmables to full brightness on the standard screen toggles as well as changing `.fastoff()` to `.off()` does respect the ramp.   So, with regard to this pull request, I'll make that change.

I do think there is still an outstanding issue which I can't put my finger on (I can make it later).  That is, even with my original issue here, I'd try overriding it by putting a `brightness=255` in a `customization:` block,  but to no affect.  That is still true.  Even with `None`, the behavior is not as would be expected.  

With `None`, if you toggle a dimmable light it goes from 0 to 255 brightness.  If you then customize the light to have a `brightness=123` and try again, you'll find that the light will actually go to full brightness but the slider (when you disclose the slider by clicking on the light name) will properly show your 123.  If you then grab the slider and move it, it dims the light to the sliders set value.  This is just like I said similarly in #4619.  The issue there must be tied to something deeper.

So, I believe it's still useful (to me at least) to amend this PR to get the brightness full when toggling the interface switches.  I can also make a new issue for the brightness customization not having an effect, if you want.  Or, would you think it's best to hold it all up until it's all fixed?